app/controllers/api/events_controller.rb

```diff
@@ -34,7 +34,10 @@ def process_incoming_event
       if event_params_check
         if upp = user_project_preference
           upp.legacy_count[create_params[:workflow]] = create_params[:count]
-          response_status = :ok if upp.save
+          if upp.save
+            Project.increment_counter :classifiers_count, upp.project_id
```

Comment:
This events controller is for legacy projects (ouroboros and older) only, i assume this is to keep counts in sync for those projects?